examples/02_decoding/plot_haxby_different_estimators.py

```diff
@@ -69,16 +69,16 @@
 # multiple cores
 svm_cv = GridSearchCV(SVC(C=1., kernel="linear"),
                       param_grid={'C': [.1, .5, 1., 5., 10., 50., 100.]},
-                      scoring='f1', n_jobs=1)
+                      scoring='f1', n_jobs=1, cv=3, iid=True)
 
 logistic_cv = GridSearchCV(LogisticRegression(C=1., penalty="l1"),
                            param_grid={'C': [.1, .5, 1., 5., 10., 50., 100.]},
-                           scoring='f1')
+                           scoring='f1', cv=3, iid=True)
```

Comment:
iid is deprecated, and will default to False. We shouldn't be putting it in an example

Reply:
Okay, I'll set it to `False`